app/services/work_history_with_breaks.rb

```diff
@@ -0,0 +1,80 @@
+class WorkHistoryWithBreaks
+  class BreakPlaceholder
+    def initialize(month_range:)
+      @month_range = month_range
+    end
+
+    def start_date
+      @month_range.first
+    end
+
+    def length
+      @month_range.count
+    end
+  end
+
+  def initialize(work_history)
+    @work_history = work_history.sort_by(&:start_date)
+    @work_history_with_breaks = []
+    @current_job = nil
+  end
+
+  def call
+    return [] if @work_history.empty?
+
+    @work_history.each { |job| @work_history_with_breaks << job_entry(job) }
```

Comment:
I think it would be more readable to keep this `@variable` local until we've finished with it, then assign? eg

```
work_history_with_breaks = []
...
@work_history_with_breaks = a_fully_populated(work_history_with_breaks)
```

Reply:
Ended up removing the need for an instance variable but there was no need 😅